### PR TITLE
Update page.mdx

### PR DESCRIPTION
**Pull request by DustinTurska**
Update supported chains

---

## PR-Codex overview
This PR updates the page content in `page.mdx` for various account abstraction services. It adds, removes, and updates service names.

### Detailed summary
- Updated service names for account abstraction
- Added new services: `DFK`, `Cyber`, `Redstone`, `Klaytn Cypress`
- Removed services: `Treasure Ruby`, `OpBNB`, `Nautilus`, `Fuse`
- Updated status for existing services

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: src/app/connect/account-abstraction/infrastructure/page.mdx [hunk 1]
```diff
@@ -49,7 +49,16 @@ With a thirdweb API key, you get access to bundler and paymaster infrastructure
 | B3                   | ❌      | ✅      |
 | Plume                | ❌      | ✅      |
 | Camp Network         | ❌      | ✅      |
-| Vanar                | ❌      | ✅      |
+| Vanar                | ✅      | ✅      |
+| DFK                  | ❌      | ✅      |
```
Comment (nkrishang) [inline src/app/connect/account-abstraction/infrastructure/page.mdx:53]: DFK mainnet is supported

File: src/app/connect/account-abstraction/infrastructure/page.mdx [hunk 2]
```diff
@@ -49,7 +49,16 @@ With a thirdweb API key, you get access to bundler and paymaster infrastructure
...
+| Form                 | ❌      | ✅      |
+| Cyber                | ✅      | ✅      |
+| Treasure Ruby        | ✅      | ✅      |
+| Redstone             | ✅      | ✅      |
+| Klaytn Cypress       | ✅      | ✅      |
+| OpBNB                | ✅      | ✅      |
+| Nautilus             | ✅      | ✅      |
```
Comment (nkrishang) [inline src/app/connect/account-abstraction/infrastructure/page.mdx:60]: For Nautilus, Fuse, OpBnB, Treasure Ruby, -- what are their testnets?

File: src/app/connect/account-abstraction/infrastructure/page.mdx (hunk 1 above)
Comment (DustinTurska) [reply src/app/connect/account-abstraction/infrastructure/page.mdx]: Updated sorry for the oversite on that

File: src/app/connect/account-abstraction/infrastructure/page.mdx (hunk 2 above)
Comment (DustinTurska) [reply src/app/connect/account-abstraction/infrastructure/page.mdx]: Removed testnets from each, I do not see where we have enabled AA for their testnets, only mainnet. Sorry about that

**Pull request merged**